packages/website/src/docs/happy-path.md

```diff
@@ -61,7 +73,7 @@ Every board has a bit of a boilerplate, and the project we just set up contains
 
 Open `src/boards/blank.ts` in your editor window and navigate to the "Blank board" board in the debugger.
```

Comment:
It's `src/blank.ts` in Replit.
Suggestion: update the Replit fork source to include the /boards/ folder ?

Reply:
It's a typo in the doc! The `src/blank.ts` is the right location.